modules/masterbar/masterbar.php

```diff
@@ -22,6 +22,52 @@ class A8C_WPCOM_Masterbar {
 	private $user_text_direction;
 	private $user_site_count;
 
+	private $links_tracks_events = array(
+		"wp-admin-bar-switch-site"                => array( "masterbar_switch_site_link_click", "/sites" ),
+		"wp-admin-bar-blog-info"                  => array( "masterbar_blog_info_link_click", "{blog_url}" ),
+		"wp-admin-bar-site-view"                  => array( "masterbar_view_site_link_click", "{blog_url}" ),
+		"wp-admin-bar-blog-stats"                 => array( "masterbar_blog_stats_link_click", "/stats/{blog_url}" ),
+		"wp-admin-bar-plan"                       => array( "masterbar_plan_link_click", "/stats/{blog_url}" ),
+		"wp-admin-bar-plan-secondary"             => array( "masterbar_plan_badge_link_click", "/plan/{blog_url}" ),
+		//my sites - manage
+		"wp-admin-bar-new-page"                   => array( "masterbar_new_page_link_click", "/pages/{blog_url}" ),
+		"wp-admin-bar-new-page-secondary"         => array( "masterbar_add_page_link_click", "/page/{blog_url}" ),
+		"wp-admin-bar-new-post"                   => array( "masterbar_new_post_link_click", "/posts/{blog_url}" ),
+		"wp-admin-bar-new-post-secondary"         => array( "masterbar_add_new_post_link_click", "/post/{blog_url}" ),
+		"wp-admin-bar-comments"                   => array( "masterbar_comments_link_click", "/comments/{blog_url}" ),
+		//my sites - personalize
+		"wp-admin-bar-themes"                     => array( "masterbar_themes_link_click", "/design/{blog_url}" ),
+		"wp-admin-bar-themes-secondary"           => array( "masterbar_themes_customize_button_click", "/design/{blog_url}" ),
+		//my sites - configure
+		"wp-admin-bar-sharing"                    => array( "masterbar_configure_sharing_link_click", "/sharing/{blog_url}" ),
+		"wp-admin-bar-users-toolbar"              => array( "masterbar_configure_people_link_click", "/people/team/{blog_url}" ),
+		"wp-admin-bar-users-toolbar-secondary"    => array( "masterbar_configure_people_add_button_click", "/wp-admin/user-new.php" ),
```

Comment:
Should we use `/people/new/{blog_url}` here instead of `/wp-admin/user-new.php`?

Reply:
![that is correct](https://user-images.githubusercontent.com/233601/30736631-6f3a34aa-9f7b-11e7-941b-a09f333c4588.gif)

Updated on f24b321101188d2bfbfe80d8e8b8fb10e49461d2.